lib/commands/general.js

```diff
@@ -296,6 +300,46 @@ commands.mobileType = async function mobileType (opts = {}) {
   return await this.adb.typeUnicode(text);
 };
 
+
+/**
+ * @typedef {Object} InstallOptions
+ * @property {boolean} allowTestPackages [false] - Set to true in order to allow test
+ *                                                 packages installation.
+ * @property {boolean} useSdcard [false] - Set to true to install the app on sdcard
+ *                                         instead of the device memory.
+ * @property {boolean} grantPermissions [false] - Set to true in order to grant all the
+ *                                                permissions requested in the application's manifest
+ *                                                automatically after the installation is completed
+ *                                                under Android 6+.
+ * @property {boolean} replace [true] - Set it to false if you don't want
+ *                                      the application to be upgraded/reinstalled
+ *                                      if it is already present on the device.
+ * @property {boolean} partialInstall [false] - Install apks partially. It is used for 'install-multiple'.
+ *                                             https://android.stackexchange.com/questions/111064/what-is-a-partial-application-install-via-adb
+ */
+
+/**
+ * @typedef {Object} InstallMultipleApksOptions
+ * @property {Array<string>} apks - The list of APKs to install. Each APK should be a path to a apk
+ *                                  or downloadable URL as HTTP/HTTPS.
+ * @property {InstallOptions} options
```

Comment:
I assume it does not work this way for docs and it is necessary to copypaste properties